packages/metro/src/DeltaBundler/types.flow.js

```diff
@@ -63,10 +65,20 @@ export type Module<T = MixedOutput> = {|
 
 export type Dependencies<T = MixedOutput> = Map<string, Module<T>>;
 
+export type TransformInputOptions = $Diff<
+  JsTransformOptions,
+  {
+    inlinePlatform: boolean,
+    inlineRequires: boolean,
+    ...
+  },
+>;
+
 export type Graph<T = MixedOutput> = {|
   dependencies: Dependencies<T>,
   importBundleNames: Set<string>,
   +entryPoints: $ReadOnlyArray<string>,
+  +transformOptions: TransformInputOptions,
```

Comment:
Let's add a comment explaining why this is here.

```suggestion
  // Unused in core but useful for custom serializers / experimentalSerializerHook
  +transformOptions: TransformInputOptions,
```